packages/web-components/src/components/checkbox/checkbox.stories.ts

```diff
@@ -144,7 +151,7 @@ export const WithAILabel = {
         <cds-checkbox>Checkbox label</cds-checkbox>
       </cds-checkbox-group>
 
-      <cds-checkbox-group legend-text="Group label">
+      <cds-checkbox-group legend-text="Group label" orientation="vertical">
```

Comment:
```suggestion
      <cds-checkbox-group legend-text="Group label">
```

I don't think this is needed here since it will be added from the property's `reflect`.